code/modules/admin/verbs/modifyvariables.dm

```diff
@@ -629,6 +629,7 @@ GLOBAL_PROTECT(VVpixelmovement)
 	if (O.vv_edit_var(variable, var_new) == FALSE)
 		to_chat(src, "Your edit was rejected by the object.")
 		return
+	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_VAR_EDIT, args)

```

Comment:
I'll be slowly converting some signals to use this method of passing in arguments because some performance testing showed it was better than passing them in manually and it also allows some interesting modification of the args list inside the signal receiver.